### ISSUE TITLE
Display broken unicode font

### ISSUE THREAD

**Issue by digiry** [esteinberg/plantuml4idea#99]
Dear,

When I wrote plantuml file, all string in korean had been rendered with it broken.
( Korean string was encoded with unicode. )

I attached my plantuml script.
Please check this issue.

```
@startuml

|Client|
|#LightBlue|Server|
|myDB|


|Client|
start
:버전확인 요청;
|Server|
:DB 정보 요청;
|myDB|
:Client 정보 반환;
end

@enduml
```

Platform:

Ubuntu 12.04
PyCharm 4.5
PlantUML integration 1.7.0
dot - graphviz version 2.26.3-10ubuntu1.2
jdk - 1.6.0_45

**Comment by krasa** [comment]
works for me on win 7
![image](https://cloud.githubusercontent.com/assets/1160875/7784088/5b2805c2-0159-11e5-9e33-89c92e6f5e67.png)

[plantuml demo works also](http://www.plantuml.com:80/plantuml/png/gt7EoKpDAwdXgb7soKpFA77AAKsj2KujAaij0ehbLheurN1nSTN0L1MN91QLS5cztjJoPa7BsvbJtypTeV1crfGtczPQSy6rMRauARnPCFNrvYq8EOX9LX1pOBAlDynuEtE6DLTgNWeN0000
)

no idea what could be wrong. maybe someone with linux could try it also...

**Comment by digiry** [comment]
This issue was resolved after PlantUML integration 1.8.

Thank you.
